.github/workflows/generators.yml

```diff
@@ -0,0 +1,91 @@
+name: generators
+
+on: workflow_dispatch
```

Comment:
Of course here you'll put a different event, *event*ually (sorry, I *had to* xD)

I'd just recommend that one of the events should be [`schedule`](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#schedule), because generators are external, and we are not really freezing, nor controlling their availability (there is not a public registry, like PyPI, npm, crates.io, ...).

Reply:
this is 20.000 miles above my skills - this will remain `workflow_dispatch` for all of my purpose ...

Reply:
Not really, it's just a `cron` schedule, it's not that complicate :)